### PR TITLE
Features

### PR DESCRIPTION
**Pull request by leonardoeng13**
[Video presentation](https://www.loom.com/share/c3d6071c3e2542598d1f852252d71cc0)

## Description

The Ruby on Rails capstone project is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category so that you can see how much money you spent and on what.

You will create a Ruby on Rails application that allows the user to:

- [x] register and log in so that the data is private to them.
- [x] introduces new transactions associated with a category.
- [x] see the money spent on each category.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,7 +104,7 @@ $ bundle exec rspec
 
 ## Authors
```
Comment (bimbolabuari) [inline README.md]: [Optional] I suggest you use Author instead of Authors as this is a solo project

File: README.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ we are gathered here today to get through this thing called LIFE."</em><strong>
   </div>
    </br>
 
-# e-wallet app
+# Budget app
```
Comment (bimbolabuari) [inline README.md:29]: Kindly deploy the application using Heroku, Netlify, or GitHub Pages, and add the link to the deployed project in the README 👍

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @leonardoeng13, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Use devise gem for authentication
- Limit access to web app resources based on authorization rules
- Use Postgres as the database for the application

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (leonardoeng13) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (leonardoeng13) [reply README.md:29]: Done.

File: README.md [hunk 3]
```diff
@@ -26,7 +26,9 @@ we are gathered here today to get through this thing called LIFE."</em><strong>
...
+
+[Live Demo](https://leobudgetapp.herokuapp.com/)
```
Comment (Dipeshtwis) [inline README.md:31]: - [ ] I just got the first page and then it gave me the error, looks like I have to send the database or something you'll need.

Screenshot from 2022-03-17 04-01-11

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @leonardoeng13, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 3 above)
Comment (leonardoeng13) [reply README.md:31]: in a real company, just a message helps and with this simple action, we're able to improve team communication.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @leonardoeng13 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**